packages/ez_hash/ez_hash.0.1.0/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+synopsis: "Hash functions: sha3, sha256, blake2b"
+description: """
+
+Hash functions: sha3, sha256, blake2b
+"""
+authors: ["Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"]
+maintainer: [
+  "David Declerck <david.declerck@ocamlpro.com>"
+  "Steven De Oliveira <steven.de-oliveira@ocamlpro.com>"
+]
+homepage: "https://github.com/OcamlPro/ocaml-solidity"
+doc: "https://OCamlPro.github.io/ocaml-solidity/sphinx"
+bug-reports: "https://github.com/OCamlPro/ocaml-solidity/issues"
+dev-repo: "git+git+https://github.com/OcamlPro/ocaml-solidity.git"
```

Comment:
this looks suspicious (curious that the linter does not complain about `git+git+https`)